mmdet3d/core/bbox/structures/coord_3d_mode.py

```diff
@@ -63,12 +60,40 @@ class Coord3DMode(IntEnum):
     DEPTH = 2
 
     @staticmethod
-    def convert(input, src, dst, rt_mat=None):
+    def mode_coord2box(coord_mode):
+        if coord_mode == Coord3DMode.LIDAR:
+            return Box3DMode.LIDAR
+        elif coord_mode == Coord3DMode.CAM:
+            return Box3DMode.CAM
+        elif coord_mode == Coord3DMode.DEPTH:
+            return Box3DMode.DEPTH
+        else:
+            raise NotImplementedError
+
+    @staticmethod
+    def mode_box2coord(box_mode):
```

Comment:
Can we use a unified dict mapping to achieve these two functions?

Reply:
Removed these two functions since `Box3DMode.LIDAR==Coord3DMode.LIDAR` is `True`